src/tox/session/__init__.py

```diff
@@ -0,0 +1,250 @@
+"""
```

Comment:
I really dislike code in `__init__.py` -- it makes it very easy to accidentally circular import, and quite annoying to find the right file in an editor 🤷‍♂️

Reply:
I rarely run into circular import problems once you have the proper abstractions, but as pointed out we can do this later.